File: Baekjoon/브루트포스/2231.py
# https://www.acmicpc.net/problem/2231

import sys
input = sys.stdin.readline

n = int(input())
ans = 0

for i in range(1, n+1):
    tmp = i
    for j in str(i):
        tmp += int(j)
    if tmp == n:
        ans = i
        break

print(ans)

File: Baekjoon/브루트포스/2231.cpp
// https://www.acmicpc.net/problem/2231

#include <iostream>
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int N;
    cin >> N;
    for (int i = 1; i < N; i++) {
        int sum = i;
        int num = i;
        while (num) {
            sum += num % 10;
            num /= 10;
        }
        if (sum == N) {
            cout << i << '\n';
            return 0;
        }
    }
    cout << 0 << '\n';

    return 0;
}